project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.4.6
```

Comment:
can you bump this up to 1.5.2 - I believe that's the latest. You will see some deprecated syntax warnings for the some of the stuff in `build.sbt`